ocs_ci/utility/utils.py

```diff
@@ -2053,7 +2053,7 @@ def get_rook_repo(branch="master", to_checkout=None):
         run_cmd(f"git checkout {to_checkout}", cwd=cwd)
 
 
-def clone_repo(url, location, branch="master", to_checkout=None):
+def clone_repo(url, location, branch="master", to_checkout=None, clone_type=None):
```

Comment:
Do we want shallow to be default?

Reply:
currently I am leaving that option to users because users has to decide which option is best fit for their use. May be in future we can default to shallow and I don't have exact word for normal clone

Reply:
As the other types of clones are typically used in development, I think `shallow` would be a fine default. I think the general use case for `clone_repo` is to download a repo/branch, run a script, then delete the repo afterwards without utilizing the commit history or trees in any way.

We could change the default to `shallow` as long as we specify in the docstring that we default to a shallow clone and a normal clone will be performed if the value is `None`. That being said, either way works for me as this is a good change.

Reply:
done